probe/endpoint/conntrack.go

```diff
@@ -463,14 +463,14 @@ func (c *conntrackWalker) handleFlow(f flow, forceAdd bool) {
 
 // walkFlows calls f with all active flows and flows that have come and gone
 // since the last call to walkFlows
-func (c *conntrackWalker) walkFlows(f func(flow)) {
+func (c *conntrackWalker) walkFlows(f func(flow, bool)) {
```

Comment:
Can you please document the new `alive` parameter? (maybe even using that name explicitly in the parameter list if Go allows it).